app/models/financial_transaction_class.rb

```diff
@@ -0,0 +1,16 @@
+class FinancialTransactionClass < ActiveRecord::Base
+  has_many :financial_transaction_types, dependent: :destroy
+
+  validates :name, presence: true
+  validates_uniqueness_of :name
+
+  scope :table_columns, -> { order(name: :asc) }
```

Comment:
I'd like to avoid default scopes, perhaps have a default `sorted` scope if you want to have a default sorting order. (Not `ordered` because we use it in a different sense elsewhere.)